src/cmd-release-ocp

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env python3
+# NOTE: PYTHONUNBUFFERED is set in the entrypoint for unbuffered output
+#
+# Publish an oscontainer as the machine-os-content in an OpenShift release
+# series.
+
+import atexit
+import argparse
+import json
+import logging as log
+import os
+import shutil
+import sys
+import tarfile
+import tempfile
+import urllib.request
+
+
+COSA_PATH = os.path.dirname(os.path.abspath(__file__))
+sys.path.insert(0, COSA_PATH)
+from cosalib.meta import GenericBuildMeta as Meta
+from cosalib.cmdlib import (
+    get_basearch,
+    run_verbose
+)
+
+
+OCP_SERVER = "https://api.ci.openshift.org"
+OCP_RELEASE_STREAM = "quay.io/openshift-release-dev/ocp-release"
+OCP_TOOL_MIRROR = "https://mirror.openshift.com/pub/openshift-v4/clients/oc/{ocp_ver}/linux/oc.tar.gz"
+
+log.basicConfig(
+    format='[%(levelname)s]: %(message)s',
+    level=log.INFO)
+
+
+def ocp_versions(meta):
+    """
+    Return the corresponding OCP version.
+    RHCOS is versioned <OCP MAJ><OCP MIN>.<RHEL MAJ><RHEL MIN>.
+    For example OCP version 4.6 with RHEL 8.1 would be 46.81
+    """
+
+    if "OpenShift" not in meta.get("summary", None):
+        return None, None
+    ostree_v = meta.get("ostree-version").split('.')
+    ocp_major, ocp_minor = list(ostree_v[0])
+    os_major, os_minor = list(ostree_v[1])
+    return (f"{ocp_major}.{ocp_minor}", f"{os_major}.{os_minor}")
+
+
+def fetch_ocp_bin(ocp_ver):
```

Comment:
It's tempting to say we should just include the latest `oc` in cosa.

Reply:
I _reallly_ thought about it. But I figured it would be better to use the versioned tool (which is what the OCP CI release tooling does).

Reply:
I personally have versioned copies of `oc` locally, i.e. `oc-4.5` and `oc-3.6` (the latter is mainly necessary because new `oc` can't talk to the ancient Centos CI at 3.6 cluster).  So we could do that.

(Maybe we can meet in the middle and skip the download if a binary is already in `$PATH`?)

Anyways OK as is too!

Reply:
Updated and dropped `--keep-bin`. It will now look for `oc-<VER>` unless you provide `--fetch-bin` or `--oc-bin`.